tests/lib/rules/spaced-comment.js

```diff
@@ -349,6 +375,52 @@ ruleTester.run("spaced-comment", rule, {
                 message: "Unexpected space or tab after /* in comment.",
                 type: "Block"
             }]
+        },
+        {
+            code: "//-----------------------\n// A comment\n//-----------------------",
+            options: ["always", {
+                block: { exceptions: ["-", "=", "*", "#", "!@#"] }
+            }],
+            errors: [
+                { message: "Expected space or tab after // in comment.", type: "Line"},
+                { message: "Expected space or tab after // in comment.", type: "Line"}
+            ]
+        },
+        {
+            code: "var a = 1; /*######*/",
+            options: ["always", {
+                line: { exceptions: ["-", "=", "*", "#", "!@#"] }
+            }],
+            errors: [{
+                message: "Expected space or tab after /* in comment.",
+                type: "Block"
+            }]
+        },
+        {
+            code: "//!< docblock style comment",
+            options: ["always", {
+                block: { markers: ["/", "!<"] }
+            }],
+            errors: [{
+                message: "Expected space or tab after // in comment.",
+                type: "Line"
+            }]
+        },
+        {
+            code: "/*!\n *comment\n */",
+            options: ["always", { line: { markers: ["!"] } }],
+            errors: [{
+                message: "Expected space or tab after /* in comment.",
+                type: "Block"
+            }]
+        },
+        {
+            code: "///--------\r\n/// test\r\n///--------\r\n/*/ blah */",
+            options: ["always", { markers: ["/"], exceptions: ["-"], block: { markers: [] } }],
+            errors: [{
+                message: "Expected exception block, space or tab after /* in comment.",
+                type: "Block"
+            }]
         }
```

Comment:
Can you add some tests that contain both line comments and block comments so you can be sure that block comment settings aren't affecting line comments and vice-versa?